apps/judicial-system/web/src/routes/Judge/Confirmation/Confirmation.spec.tsx

```diff
@@ -1,17 +1,73 @@
 import React from 'react'
 import { render, waitFor, screen } from '@testing-library/react'
 import { Confirmation } from './Confirmation'
-import { CaseAppealDecision } from '@island.is/judicial-system/types'
+import {
+  CaseAppealDecision,
+  UpdateCase,
+} from '@island.is/judicial-system/types'
 import { MemoryRouter, Route } from 'react-router-dom'
 import * as Constants from '../../../utils/constants'
 import {
   mockCaseQueries,
   mockJudgeQuery,
+  mockUpdateCaseMutation,
 } from '@island.is/judicial-system-web/src/utils/mocks'
 import { MockedProvider } from '@apollo/client/testing'
 import { UserProvider } from '@island.is/judicial-system-web/src/shared-components/UserProvider/UserProvider'
+import userEvent from '@testing-library/user-event'
 
 describe('Confirmation route', () => {
+  test(`should not allow users to continue unless every required field has been filled out`, async () => {
+    // Arrange
+
+    // Act
+    render(
+      <MockedProvider
+        mocks={[
+          ...mockCaseQueries,
+          ...mockJudgeQuery,
+          ...mockUpdateCaseMutation([
+            {
+              courtStartTime: '2020-09-16T15:55:000Z',
```

Comment:
Hmm, this is not correct 🤔

This should be `courtEndTime`. Could you please see if the test runs like that?

Reply:
The test also succeeds with courtEndTime set and I have pushed that change. Note however that this change will not be merged to the main branch, so if it is also wrong there, then it needs to be fixed there as well.